tests/bridge_erc20.ts

```diff
@@ -65,60 +37,45 @@ describe('Bridging ERC20 tokens', async function () {
 
   before(async () => {
     env = await setupEnvironment({});
+    eth_testToken = await getOrDeployECR20Contract(env);
+    eth_testTokenAddress = eth_testToken.address.toLowerCase();
+    fuel_testToken = await getOrDeployFuelTokenContract(env, eth_testToken, FUEL_TX_PARAMS);
+    fuel_testTokenId = fuel_testToken.id.toHexString();
   });
 
   it('Setup tokens to bridge', async () => {
-    // TODO: use config time values in sway contracts so we don't have to hardcode
-    // these values and can create a new test token contract each time
-    const expectedGatewayContractId = '0x5FC8d32690cc91D4c39d9d3abcBD16989F875707';
-    const expectedTokenContractId = '0x0165878A594ca255338adfa4d48449f69242Eb8F';
-
-    // create test ERC20 contract
-    try {
-      eth_testToken = Token__factory.connect(expectedTokenContractId, env.eth.deployer);
-      await eth_testToken.totalSupply();
-    } catch (e) {
-      const eth_tokenFactory = new Token__factory(env.eth.deployer);
-      eth_testToken = await eth_tokenFactory.deploy();
-      await eth_testToken.deployed();
-    }
+    const { value: expectedTokenContractId } = await fuel_testToken.functions.bridged_token().get();
 
     // check that values for the test token and gateway contract match what
     // was compiled into the bridge-fungible-token binaries
-    expect(env.eth.fuelERC20Gateway.address).to.equal(expectedGatewayContractId);
-    expect(eth_testToken.address).to.equal(expectedTokenContractId);
+    expect(fuel_to_eth_address(expectedTokenContractId)).to.equal(eth_testTokenAddress);
```

Comment:
Is the gateway no longer being checked here?

Reply:
I just added back this test but getting it from the contract. The way it was built before was not checking much this is why I removed.